src/meta/proto-conv/src/user_from_to_protobuf_impl.rs

```diff
@@ -494,11 +494,18 @@ impl FromToProto for mt::FileFormatOptions {
             })?,
         )?;
 
+        let nan_display = if p.nan_display.is_empty() {
```

Comment:
why set to "nan"?

Reply:
if need this,  should be done at a higher level and format by format. the check and set default logic is mainly in file_format_type.rs now,  which may need  refactor later too.